config/HELICSConfig.cmake.in

```diff
@@ -85,6 +69,20 @@ endif()
 
 check_required_components(${PN})
 
+get_target_property(${PN}_INCLUDE_DIRS HELICS::helics_base_includes INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
Does this include the directory with the HELICS headers?  I wasn't sure if helics_base_includes actually did that as I thought it was for all the other includes.

Reply:
It looks like it does -- the installed targets file has the include directory with the HELICS headers, it looks like CMakeLists.txt:415 is setting it.